skiko/src/commonMain/kotlin/org/jetbrains/skiko/SkikoView.kt

```diff
@@ -7,6 +7,7 @@ interface SkikoView {
     fun onKeyboardEvent(event: SkikoKeyboardEvent) = Unit
     fun onPointerEvent(event: SkikoPointerEvent) = Unit
     fun onInputEvent(event: SkikoInputEvent) = Unit
+    fun onTouchEvent(events: Set<SkikoTouchEvent>) = Unit
```

Comment:
Can we use less resource consuming structure, i.e. `Array<SkikoTouchEvent>`?

Reply:
Done. Made it `Array`